src/mocks/mockUser.ts

```diff
@@ -1,7 +1,7 @@
 import { IUser } from '@/interfaces/userInterfaces';
 
 const mockUser: IUser = {
-  id: '9d6a0854-a784-44b0-ad35-ca5e8b8f0e90',
+  id: '9990de74-96ac-4896-bd5f-f4bb506da38f',
```

Comment:
Why this value has changed? Checking the Add to favourite star breaks the save operation

Reply:
This whole operation needs to be changed to work with the current person from our API /persons. I forgot to create a thunk to fetch the user. The operation breaks, since the mock user doesn't exist in our database.